colors/spaceduck.vim

```diff
@@ -23,7 +23,31 @@ let g:colors_name="spaceduck"
 
 " Spaceduck Color Variables {{{
 " dark theme and light theme settings
-" TODO light theme isn't done yet
+let s:palette = {
+      \ 'red':          ['#e33400', '166'],
+      \ 'orange':       ['#e39400', '172'],
+      \ 'green':        ['#5ccc96', '78'],
+      \ 'yellow':       ['#f2ce00', '220'],
+      \ 'lavender':     ['#b3a1e6', '146'],
+      \ 'grape':        ['#7a5ccc', '98'],
```

Comment:
hahah I love that you took the suggested colors from #28 !

Reply:
Yeah I read the issue and thought it was a good idea. 😃